### PR TITLE
feat: colors page

### PR DESCRIPTION
**Pull request by agazso**
Adds a simple page which checks the CSS colors periodically and prints them in a code block, so that it can be copied in the CSS file.

### CODE REVIEW

File: src/routes/colors/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		if (!browser) {
+			return ''
+		}
+		return document?.documentElement?.style?.getPropertyValue(name)
+	}
+	let css = $state('')
+
+	onMount(() => {
+		setInterval(() => getColors(), 500)
+	})
```
Comment (vojtechsimetka) [inline src/routes/colors/+page.svelte:16]: This is never cleared

**Review by vojtechsimetka — COMMENTED** [review]
Have you considered using [MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver)?

Should this page be linked from the menu?

**Comment by agazso** [comment]
> Have you considered using [MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver)?

Yes, but it looks to me that using that for computed styles can be quite complicated: https://stackoverflow.com/questions/20364687/mutationobserver-and-current-computed-css-styles

> Should this page be linked from the menu?

I planned to link to here from the build page (#230) but we can add it to the menu as well. @david-gauquelin wdyt?

**Review by vojtechsimetka — APPROVED** [review]
I think you may be right. Feel free to merge.

Just for the sake of how this could look like (thanks ChatGPT):
```ts
   onMount(() => {
        if (browser) {
            const observer = new MutationObserver((mutations) => {
                mutations.forEach((mutation) => {
                    if (mutation.type === 'attributes' && mutation.attributeName && mutation.target === document.documentElement) {
                        getColors()
                    }
                })
            })

            observer.observe(document.documentElement, {
                attributes: true,
                attributeFilter: [
                    'style'
                ]
            })

            // Initialize colors on mount
            getColors()

            onDestroy(() => {
                observer.disconnect()
            })
        }
    })
```

**Comment by agazso** [comment]
> I think you may be right. Feel free to merge.
> 
> Just for the sake of how this could look like (thanks ChatGPT):
> 
> ```ts
>    onMount(() => {
>         if (browser) {
>             const observer = new MutationObserver((mutations) => {
>                 mutations.forEach((mutation) => {
>                     if (mutation.type === 'attributes' && mutation.attributeName && mutation.target === document.documentElement) {
>                         getColors()
>                     }
>                 })
>             })
> 
>             observer.observe(document.documentElement, {
>                 attributes: true,
>                 attributeFilter: [
>                     'style'
>                 ]
>             })
> 
>             // Initialize colors on mount
>             getColors()
> 
>             onDestroy(() => {
>                 observer.disconnect()
>             })
>         }
>     })
> ```

Based on this I managed to write the code with MutationObserver

**Pull request merged**